ocp_resources/ingress.py

```diff
@@ -0,0 +1,5 @@
+from ocp_resources.resource import Resource
+
+
+class Ingress(Resource):
```

Comment:
Ingress is a NamespacedResource

Reply:
it's not from what i tried (i tried the oc command first, obviously, without namespace). if it should be changed, a bug will be in order.

i was provided the command and had to figure out by myself how to use the resource to fetch it.